src/support/command/parse-input.js

```diff
@@ -70,7 +70,7 @@ const readConfigFile = filename => {
 
 const splitPathParams = path => path.split('/').filter(Boolean)
 
-const splitFlagParams = flag => flag.split(',').filter(Boolean)
+const splitFlagParams = flag => flag.replace(/\s+/g, '').split(',').filter(Boolean)
```

Comment:
Could you split, then trim()?

Reply:
Probably would be a bit more readable for the next person who looks at it, I'll change it to that